dev/AppLifecycle/Association.cpp

```diff
@@ -461,4 +461,9 @@ namespace winrt::Microsoft::Windows::AppLifecycle::implementation
             }
         }
     }
+
+    void NotifyShellAssocChanged()
+    {
+        SHChangeNotify(SHCNE_ASSOCCHANGED, SHCNF_IDLIST, 0, 0);
```

Comment:
note, this is a very expensive notification, causing the taskbar to re-pain on Windows 10 and below. use it carefully.

Reply:
the last 2 params are pointers, thus `nullptr`